lib/web_ui/lib/src/engine/canvaskit/embedded_views.dart

```diff
@@ -424,7 +413,11 @@ class HtmlViewEmbedder {
             ? null
             : diffViewList(_activeCompositionOrder, _compositionOrder);
     _updateOverlays(diffResult);
-    assert(_context.pictureRecorders.length == _overlays.length);
+    assert(
+      _context.pictureRecorders.length == _overlays.length,
+      'There should be the same picture recorders (${_context.pictureRecorders.length}) '
+      'than overlays (${_overlays.length}).',
```

Comment:
I'm finding having `and` as opposed to `than` in the assert makes it easier to read and understand.

Reading `than`, makes me suspect that the assert is supposed to assert that:

- `_context.pictureRecorders.length` is supposed to be greater/lower **`than`** `_overlays.length`
- and not that `_context.pictureRecorders.length` and `_overlays.length` should be the same. _(the intended)_

Makes sense?

Reply:
I quickly ran grammarly through this, and it definitely prefers "same number of" instead of just "same" (I lost energy typing there), and it doesn't care about "as" or "and", but it does flag "than" as wrong, so

"There should be the same number of picture recorders (5) as overlays (7)." it is!